packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -34,7 +34,7 @@
     "@types/mocha": "^5",
     "@types/node": "12.0.12",
     "@types/sinon": "^2.3.2",
-    "@types/vscode": "1.40.0",
+    "@types/vscode": "1.46.0",
```

Comment:
should we have the caret for @types as well since it's there for the engine version?

Reply:
I didn't add it for the types since those can change and I want us to use the lowest supported version types, in this case version 1.46.0.